pkg/distsql/context/context.go

```diff
@@ -86,8 +87,8 @@ type DistSQLContext struct {
 
 	ExecDetails *execdetails.SyncExecDetails
 
-	// Only one cop-reader can use lite worker. Using lite-worker in multiple readers will affect the concurrent execution of readers.
-	TryCopLiteWorker uint32
+	// Only one cop-reader can use lite worker at the same time. Using lite-worker in multiple readers will affect the concurrent execution of readers.
+	TryCopLiteWorker atomic.Uint32
```

Comment:
Can't use `sync/atomic` here, since

```sh
nogo: errors found by nogo during build-time code analysis:

pkg/distsql/context/context.go:105:12: assignment copies lock value to newCtx: github.com/pingcap/tidb/pkg/distsql/context.DistSQLContext contains sync/atomic.Uint32 contains sync/atomic.noCopy (copylocks)
```